package.json

```diff
@@ -1,11 +1,24 @@
 {
   "name": "derivable",
-  "version": "0.12.1",
+  "version": "0.13.0-beta7",
   "description": "Functional Reactive State for JavaScript & TypeScript",
   "author": "David Sheldrick",
   "files": [
     "dist"
   ],
+  "scripts": {
+    "build": "node scripts/build.js",
+    "test": "./node_modules/.bin/mocha --recursive",
+    "bench": "node scripts/bench.js",
+    "coverage": "./scripts/coverage.sh && ./node_modules/.bin/istanbul report --include=coverage-final.json text",
```

Comment:
./node_modules/.bin is on the path when doing npm run.
so you can make it shorter like this:
"test": "mocha --recursive"